### PR TITLE
Add support for Devuan-based OSes

### PR DESCRIPTION
**Pull request by linuxitux**
Tested on Devuan 1.0 Jessie.

### CODE REVIEW

**Comment by SwartzCr** [comment]
Associated with #4187, but maybe they should be combined into one PR?

**Comment by linuxitux** [comment]
@SwartzCr sorry about that.

**Comment by bmw** [comment]
Is Certbot available from [jessie-backports](https://devuan.org/os/etc/apt/sources.list.d/devuan-stable-backports.list) on Devuan? We're actively trying to deprecate `certbot-auto` and OS packages should be certainly preferred so we'd like not to add support for more OSes to the script for systems where there are better options available.

**Comment by linuxitux** [comment]
@bmw yes, it is available from jessie-backports on Devuan.

**Comment by bmw** [comment]
In this case, I'm closing this PR due to my message above. `certbot-auto` is disproportionately the cause of many of the problems people have with Certbot and OS packages should certainly be preferred. I hope this makes sense and I'm sorry for the miscommunication here!

**Comment by linuxitux** [comment]
@bmw Alright. I must say this came up because I followed the official instructions posted on:

https://certbot.eff.org/#debianwheezy-other

Thank you anyway.

**Comment by linuxitux** [comment]
@bmw Now I see the tutorial it's different for "Jessie":

https://certbot.eff.org/#debianjessie-other

**Comment by SwartzCr** [comment]
No worries! Thanks for bringing it up!

File: certbot-auto [hunk 1]
```diff
@@ -591,6 +591,9 @@ Bootstrap() {
   elif [ -f /etc/debian_version ]; then
     BootstrapMessage "Debian-based OSes"
     BootstrapDebCommon
+  elif [ -f /etc/devuan_version ]; then
```
Comment (nuxwin) [inline certbot-auto]: well done ;)

**Comment by nuxwin** [comment]
@bmw 

Following our discusion on IRC, could it be possible to merge that PR, at least for the time being? Tested with Devuan Jessie 1.0 (RC). Work same as under Debian Jessie.

Thanks.

**Pull request closed without merging**